cluster/templates/fluo-cluster/conf/yarn-site.xml

```diff
@@ -53,4 +53,24 @@
     <name>yarn.nodemanager.delete.debug-delay-sec</name>
     <value>259200</value>
   </property>
+  <property>
+    <name>yarn.nodemanager.container-executor.class</name>
+    <value>org.apache.hadoop.yarn.server.nodemanager.LinuxContainerExecutor</value>
+  </property>
+  <property>
+    <name>yarn.nodemanager.container-executor.resources-handler.class</name>
+    <value>org.apache.hadoop.yarn.server.nodemanager.util.CgroupsLCEResourcesHandler</value>
+  </property>
+  <property>
+    <name>yarn.nodemanager.linux-container-executor.path</name>
+    <value>/hadoop-ce/bin/container-executor</value>
```

Comment:
> I had to place the hadoop-ce directory in the linux root dir as the Linux container executor has funky permission requirements that need to be applied to all parent directories. I didn't want to apply those permissions to /opt or /var/local/

Thats extremely useful info.    IMO it would be nice to add that info as a comment in the xml config.